src/app/tests/suites/certification/Test_TC_CHANNEL_5_9.yaml

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 19.7.2. [TC-CHANNEL-5.9] Change Channel Verification (DUT as Server)
+
+PICS:
+    - CHANNEL.S
+
+config:
+    nodeId: 0x12344321
+    cluster: "Channel"
+    endpoint: 1
+
+tests:
+    - label: "Wait for the commissioned device to be retrieved"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "Step 1: TH reads the ChannelList attribute from the DUT"
+      PICS: CHANNEL.S.A0000
+      command: "readAttribute"
+      attribute: "ChannelList"
+      response:
+          constraints:
+              type: list
+
+    - label:
+          "Step 2: TH sends a ChangeChannel command to the DUT with a channel
+          name from the list in step 1"
+      PICS: CHANNEL.S.C00.Rsp
+      command: "ChangeChannel"
+      arguments:
+          values:
+              - name: "Match"
+                value: "PBS" # Example, choose from the available ChannelList
+
+    - label: "Step 2a: Verify that the channel has changed on the device"
+      cluster: "LogCommands"
+      command: "UserPrompt"
```

Comment:
we generally disallow manual tests in the harness. This does seem optional though as we check `current channel`, however I wonder if we should adjust the test plan instead:
 - step 2 should probably pick a "different channel than the current one"
 - step 3 can verify that channel change was successful ... maybe reading the new channel name is sufficient
 
 If we can run tests without human intervention, that would be better (and in this case if we can trust `CurrentChannel` that would be great).